src/main/java/org/salespointframework/order/OrderManagement.java

```diff
@@ -93,8 +94,8 @@
 
 	/**
 	 * Returns all {@link Order}s from the given {@link UserAccount} in between the dates {@code from} and {@code to},
-	 * including from and to. So every entry with an time stamp <= to and >= from is returned. If this user has no
-	 * {@link Order}s in this period, an empty {@link Iterable} is returned.
+	 * including from and to. So every entry with an time stamp {@literal <=} to and {@literal >=} from is returned.
```

Comment:
```suggestion
	 * including from and to. So every entry with an time stamp {@literal &lt;=} to and {@literal &gt;=} from is returned.
```

Reply:
I now implemented the requested changes, however i used &le;/&ge; instead of &lt;/&gt;